tests/complexity/scripts/ex_assembly.py

```diff
@@ -50,11 +50,11 @@ def assemble_matrix_ex01(ne1: 'int', ne2: 'int',
                             v = 0.0
                             for g1 in range(0, k1):
                                 for g2 in range(0, k2):
-                                    bi_0 = basis_1[ie1, il_1, 0, g1] * basis_2[ie2, il_2, 0, g2]
+                                    # bi_0 = basis_1[ie1, il_1, 0, g1] * basis_2[ie2, il_2, 0, g2]
```

Comment:
I think this has changed your computational intensity results. Do you not need `bi_0`/`bj_0`? Do you think it makes sense to change the expected intensity or are these here for a reason? We could just return them to avoid them being seen as useless?

Reply:
I think as long as it is just a function we are using for testing purpose it would be better to delete them and edit the complexity value

Reply:
Ok. Then should we delete them rather than just commenting?